ansible/roles/rhacm_assisted_installer/files/assisted-installer.yml

```diff
@@ -0,0 +1,37 @@
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: assisted-installer
+  namespace: assisted-installer
+spec:
+  targetNamespaces:
+  - assisted-installer
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  labels:
+    operators.coreos.com/assisted-service-operator.assisted-installer: ""
+  name: assisted-service-operator
+  namespace: assisted-installer
+spec:
+  channel: alpha
+  config:
+    env:
+    - name: OPENSHIFT_VERSIONS
+      value: '{"4.8":{"display_name":"4.8.0-fc.0","release_image":"quay.io/openshift-release-dev/ocp-release:4.8.0-fc.0-x86_64","rhcos_image":"https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.7/4.7.0/rhcos-4.7.0-x86_64-live.x86_64.iso","rhcos_version":"47.83.202102090044-0","support_level":"beta"}}'
+    - name: INSTALLER_IMAGE
+      value: quay.io/ocpmetal/assisted-installer:latest
```

Comment:
Where does this code run? Could it have u/s images?
Anyway, it should use fixed versions.

Reply:
We are currently running in the scale lab.  What exactly is a "u/s" image?  For the bare metal hub we use image version `v1.0.18.3` but I am unsure if that includes what is required at the moment for acm hub using hive/ai to provision sno clusters.